boards/native/Makefile.dep

```diff
@@ -19,3 +19,9 @@ ifneq (,$(filter socket_zep,$(USEMODULE)))
   USEMODULE += checksum
   USEMODULE += random
 endif
+
+ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
```

Comment:
This duplicates the `netdev` line from above, pulling in `netdev_raw802154` and `netdev_tap` always.